### PR TITLE
leptonica: Upgrade to 1.18.1

### PR DESCRIPTION
**Pull request by jannick0**
@lazka Ok that the CLANG32 dependencies are not available? Or are they to be regenerated?

### CODE REVIEW

File: mingw-w64-leptonica/PKGBUILD [hunk 1]
```diff
@@ -29,28 +29,25 @@ prepare() {
...
+  test -d "${_builddir}" && rm -rf "${_builddir}"
+  mkdir -p "${_builddir}" && cd "${_builddir}"
   CFLAGS+=" -DMINIMUM_SEVERITY=L_SEVERITY_WARNING"
   ../${_realname}-${pkgver}/configure -C \
-    --disable-dependency-tracking \
-    --disable-silent-rules \
-    --build=${MINGW_CHOST} \
-    --host=${MINGW_CHOST} \
-    --target=${MINGW_CHOST} \
-    --prefix=${MINGW_PREFIX}
```
Comment (lazka) [inline mingw-w64-leptonica/PKGBUILD:41]: why these changes?

File: mingw-w64-leptonica/PKGBUILD [hunk 2]
```diff
@@ -29,28 +29,25 @@ prepare() {
...
-  make
+    --disable-dependency-tracking
+  make V=0 || make V=1
```
Comment (lazka) [inline mingw-w64-leptonica/PKGBUILD:38]: does the build fail the first time?

**Comment by lazka** [comment]
> @lazka Ok that the CLANG32 dependencies are not available? Or are they to be regenerated?

yes, please ignore

File: mingw-w64-leptonica/PKGBUILD (hunk 1 above)
Comment (jannick0) [reply mingw-w64-leptonica/PKGBUILD:41]: These are set by `makepkg`, except for `--build` and `--target` which are redundant and only needed if cross-compiling and compiling compilers, resp.

Happy to add back
```
--host=${MINGW_CHOST}
--prefix=${MINGW_PREFIX}
```
the other stuff should go as in the commit.

File: mingw-w64-leptonica/PKGBUILD (hunk 2 above)
Comment (jannick0) [reply mingw-w64-leptonica/PKGBUILD:38]: No, but after it failed in non-verbose mode, `make` is re-run to show the failing command line.  Droppable if you prefer. Either way I am fine. Locally I am running things as given in the commit.

Comment (lazka) [reply mingw-w64-leptonica/PKGBUILD:38]: ok, we've had this pattern when "make" had missing dependencies, or crashes which could be worked around by running it twice. This means I'd be afraid to change it in the future without knowing your motivation.

But I guess this makes sense.

File: mingw-w64-leptonica/PKGBUILD (hunk 1 above)
Comment (lazka) [reply mingw-w64-leptonica/PKGBUILD:41]: ok, thanks, makes sense. We likely use it everywhere because prior to https://github.com/msys2/MSYS2-packages/issues/2285 CONFIG_SITE wasn't always set. But that's no longer the case now.

Comment (lazka) [reply mingw-w64-leptonica/PKGBUILD:41]: I'm not sure if `--prefix=${MINGW_PREFIX}` still has value for consistency and documentation, since we need to set it with all other build tools.

Comment (jannick0) [reply mingw-w64-leptonica/PKGBUILD:41]: to be on the safe side let me add back the defs of `prefix` and `host` ... together with a simple call of `make`.  Then everything is golden I guess.

Comment (lazka) [reply mingw-w64-leptonica/PKGBUILD:41]: We are definitely cargo culting various things for no good reason :) I'm fine with removing host if you think it's not needed. You definitely know much more about autotools then me

Comment (lazka) [reply mingw-w64-leptonica/PKGBUILD:41]: > together with a simple call of `make`

or add a small comment maybe `# call make again to see debug output if the first fails for some reason`

Comment (jannick0) [reply mingw-w64-leptonica/PKGBUILD:41]: All good - thanks. ... Force-pushed new more explicit version ready to be merged once CI is happy.
